extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -146,6 +144,18 @@ export const selectDatabase = localize('selectDatabase', "Select database");
 export const done = localize('done', "Done");
 export const nameMustNotBeEmpty = localize('nameMustNotBeEmpty', "Name must not be empty");
 
+// Publish Dialog options
+export const publishOptions = localize('publishOptions', 'Publish Options');
+export const configureOptions = localize('configureOptions', 'Configure Options');
+export const GeneralOptions: string = localize('generalOptions', "General Options");
+export const OkButton: string = localize('ok', "Ok");
+export const CancelButton: string = localize('cancel', "Cancel");
```

Comment:
Reason for created a new one is, since the existing ones are using for publish dialog window, thinking if the button text changes in future and if someone changes the existing one which updates the references unwantedly. But yeah using the existing ones, since the name also needs to be changed if want to and that gives the references updates. Thanks.